config.yaml

```diff
@@ -1,5 +1,5 @@
 database:
-  url: postgres://postgres:s3cr3t@localhost:5432/barad-dur
```

Comment:
Why did you remove the database name and change the secret and such?

Reply:
Woops, the point was to remove `config.yaml` completely, I think it was added by accident at some point in the past since it's in `.gitignore`. Like everywhere we have only `config.sample.yaml` in repos, and not `config.yaml` themselves

Reply:
Makes sense, I think you should just delete config.yaml